not-published/android/using-auth-sdk.adoc

```diff
@@ -335,7 +335,18 @@ authService.logout(currentUser, new Callback<UserPrincipal>() {
 By default, the local tokens obtained during authentication are only deleted when the logout succeeded against the authentication server.
 You can use the `AuthService#deleteTokens` function to delete the local authentication tokens as part of a failed logout, or for other use cases.
 
-*Note:* To perform backchannel or federated logouts, you must enable the Backchannel Logout option for the federated identity provider. More information is available in the Keycloak documentation under  http://www.keycloak.org/docs/latest/server_admin/index.html#openid-connect-v1-0-identity-providers[OIDC Identity Providers].
+NOTE: To perform backchannel or federated logouts, you must enable the Backchannel Logout option for the federated identity provider. More information is available in the Keycloak documentation under  http://www.keycloak.org/docs/latest/server_admin/index.html#openid-connect-v1-0-identity-providers[OIDC Identity Providers].
+
+=== Using auth with backend
+
+To authenticate service on existing backend server users need to supply additional token:
+
+[source]
+----
+Authorization: BEARER Yourtoken
+----
+
+Auth SDK also provides _AuthHeaderProvider_ class that can be used for this purpose.
```

Comment:
Maybe we should add some code example?

```java
MobileCore.getInstance().getHttpLayer().requestHeaderInterceptor().add(yourAuthHeaderProviderInstance);
```

NOR

We say we have a util method on authService to help you on it

```java
﻿MobileCore.getInstance().getHttpLayer().requestHeaderInterceptor().add(authService.getAuthHeaderProvider());
```